examples/nlp/gpt/train_gpt_dpo.py

```diff
@@ -107,6 +117,7 @@ def main(cfg) -> None:
         collate_fn=identity_collate,
     )
 
+    # collate = validation_ds.collate_fn if cfg.model.data.data_impl=="packed_jsonl" else dpo_custom_collate
```

Comment:
can this be removed?

Reply:
Yes, but note that using the same collate fn for the train and validation datasets means that the datasets must have the same `seq_length` and `tokenizer`.